bn_mp_to_sbin.c

```diff
@@ -1,15 +1,18 @@
 #include "tommath_private.h"
-#ifdef BN_MP_TO_SIGNED_BIN_C
+#ifdef BN_MP_TO_SBIN_C
 /* LibTomMath, multiple-precision integer library -- Tom St Denis */
 /* SPDX-License-Identifier: Unlicense */
 
 /* store in signed [big endian] format */
-mp_err mp_to_signed_bin(const mp_int *a, unsigned char *b)
+mp_err mp_to_sbin(const mp_int *a, unsigned char *b, size_t maxlen, size_t *written)
 {
    mp_err err;
-   if ((err = mp_to_unsigned_bin(a, b + 1)) != MP_OKAY) {
+   if ((err = mp_to_ubin(a, b + 1, maxlen, written)) != MP_OKAY) {
```

Comment:
maxlen - 1?

Reply:
> maxlen - 1?

Yeah, added the sign to `written` but forgot to subtract it from the `maxlen`. Will correct.

>  (also check for maxlen==0).

Why? Ah, yes, `maxlen` is `size_t`, right. Will correct.